README.md

```diff
@@ -12,63 +12,63 @@ This CDK library automatically configures ingestion of metrics, traces, and logs
 - Enabling the collection of traces and custom metrics from your Lambda functions.
 - Managing subscriptions from the Datadog Forwarder to your Lambda function log groups.
 
-## Installation
-
+## Npm Package Installation:
 ```
 yarn add --dev datadog-cdk-constructs
 # or
 npm install datadog-cdk-constructs --save-dev
 ```
 
+## PyPi Package Installation:
```

Comment:
Nit: `PyPi` -> `PyPI`

Reply:
(I would make sure this is correct throughout the PR, there are some other instances of this)